server/routes/visitJourney/index.ts

```diff
@@ -1,23 +1,22 @@
 import { type RequestHandler, Router } from 'express'
-import sessionCheckMiddleware from '../middleware/sessionCheckMiddleware'
-import asyncMiddleware from '../middleware/asyncMiddleware'
-import SelectVisitors from './visitJourney/selectVisitors'
-import VisitType from './visitJourney/visitType'
-import DateAndTime from './visitJourney/dateAndTime'
-import CheckYourBooking from './visitJourney/checkYourBooking'
-import Overbooking from './visitJourney/overbooking'
-import Confirmation from './visitJourney/confirmation'
-import AdditionalSupport from './visitJourney/additionalSupport'
-import MainContact from './visitJourney/mainContact'
-import RequestMethod from './visitJourney/requestMethod'
-import type { Services } from '../services'
+import sessionCheckMiddleware from '../../middleware/sessionCheckMiddleware'
+import asyncMiddleware from '../../middleware/asyncMiddleware'
+import SelectVisitors from './selectVisitors'
+import VisitType from './visitType'
+import DateAndTime from './dateAndTime'
+import CheckYourBooking from './checkYourBooking'
+import Overbooking from './overbooking'
+import Confirmation from './confirmation'
+import AdditionalSupport from './additionalSupport'
+import MainContact from './mainContact'
+import RequestMethod from './requestMethod'
+import type { Services } from '../../services'
+import { BookOrUpdate } from '../../@types/bapv'
 
-export default function routes({
-  auditService,
-  prisonerVisitorsService,
-  visitService,
-  visitSessionsService,
-}: Services): Router {
+export default function routes(
```

Comment:
This single function can now set up both book and update routes - so we don't have the duplication (and risk of missing a new page by adding it to only book or update!)